lstchain/reco/utils.py

```diff
@@ -201,17 +201,18 @@ def get_event_pos_in_camera(event, tel):
     -------
     (x, y) (float, float): position in the camera
     """
-    array_pointing = HorizonFrame(alt=event.mcheader.run_array_direction[1],
+    array_pointing = AltAz(alt=event.mcheader.run_array_direction[1],
```

Comment:
The `array_pointing' should be defined in this way
```python
from astropy.coordinates import SkyCoord, EarthLocation, AltAz
from astropy.time import Time

location = EarthLocation.of_site('Roque de los Muchachos')
obstime = Time('2018-11-01T02:00')
horizon_frame = AltAz(location=location, obstime=obstime) 
array_pointing = SkyCoord(az=event.mcheader.run_array_direction[0],
                          alt=event.mcheader.run_array_direction[1], 
                          frame=horizon_frame)
```
so that at first you define the frame and then specify the coordinates in that frame.